GVFS/GVFS.Platform.Mac/MacFileSystemVirtualizer.cs

```diff
@@ -99,7 +99,6 @@ public override void Stop()
                     relativePath,

```

Comment:
Will you add a `summary` to this method indicating that the `endOfFile` parameter is not used?